src/PerfView/PerfViewData.cs

```diff
@@ -493,6 +498,38 @@ public override void Open(Window parentWindow, StatusBar worker, Action doAfter
                     m_singletonStackSource.Viewer.Focus();
             }
         }
+
+        public void OpenOnExistingWorker(Action doAfter = null)
```

Comment:
Can you describe the difference between OpenOnExistingWorker() and Open() and the scenarios where it is used and Open() does not work?    At the least we need to document it, but I would rather try to fold it into Open() somehow.